test/lib/sstable_utils.hh

```diff
@@ -213,6 +213,8 @@ public:
     }
 
     void set_values(sstring first_key, sstring last_key, stats_metadata stats) {
+        _sst->_data_file_size = 1;
+        _sst->_bytes_on_disk = 1;
```

Comment:
This too should be separated out and explained.
It seems to be valid even without compaction_groups.

Reply:
Yes, they can go into a separate patch. thanks